lib/AfricasTalking/Airtime.rb

```diff
@@ -1,75 +1,75 @@
+# frozen_string_literal: true
+
 class Airtime
-	include AfricasTalking
-	HTTP_CREATED     = 201
-	HTTP_OK          = 200
+  include AfricasTalking
+  HTTP_CREATED     = 201
+  HTTP_OK          = 200
+
+  # Set debug flag to to true to view response body
+  def initialize(username, apikey)
+    @username    = username
+    @apikey      = apikey
+  end
 
-	#Set debug flag to to true to view response body
-	def initialize username, apikey
-		@username    = username
-		@apikey      = apikey
-	end
+  def send(options)
+    url = getAirtimeUrl + '/send'
 
-	def send options
-		url      = getAirtimeUrl() + "/send"
+    recipients = options.each  do |item|
+      validateParamsPresence? item, %w[phoneNumber currencyCode amount]
+      item['amount'].to_s.prepend(item['currencyCode'].to_s + ' ')
+      item.delete('currencyCode')
+    end
+    post_body = {
+      'username' => @username,
+      'recipients' => recipients.to_json
+    }
+    response = sendNormalRequest(url, post_body)
+    if @response_code == HTTP_CREATED
+      responses = JSON.parse(response, quirky_mode: true)
+      if !responses['responses'].empty?
+        results = responses['responses'].collect do |result|
+          AirtimeResponse.new result['status'], result['phoneNumber'], result['amount'], result['requestId'], result['errorMessage'], result['discount']
+        end
+        return SendAirtimeResult.new responses['errorMessage'], responses['numSent'], responses['totalAmount'], responses['totalDiscount'], results
+      else
+        raise AfricasTalkingException, responses['errorMessage']
+      end
+      raise AfricasTalkingException, response
+    end
+  end
 
-		recipients = options.each{|item| 
-			validateParamsPresence? item, %w(phoneNumber currencyCode amount)
-			item['amount'].to_s.prepend(item['currencyCode'].to_s + " ") 
-			item.delete('currencyCode')
-		}
-		post_body = {
-					'username'   => @username,
-					'recipients' => recipients.to_json,
-				}
-		response = sendNormalRequest(url, post_body) 
-		if (@response_code == HTTP_CREATED)
-			responses = JSON.parse(response, :quirky_mode =>true)
-			if (responses['responses'].length > 0)
-				# 
-				results = responses['responses'].collect{ |result|
-					# 
-					AirtimeResponse.new result['status'], result['phoneNumber'],result['amount'],result['requestId'], result['errorMessage'], result['discount']
-				}
-				# 
-				return SendAirtimeResult.new responses["errorMessage"], responses["numSent"], responses["totalAmount"], responses["totalDiscount"], results
-			else
-				raise AfricasTalkingException, responses['errorMessage']
-			end
-			raise AfricasTalkingException, response
-		end
-	end
+  private
 
-	private
-		def getAirtimeUrl()
-			return getApiHost() + "/version1/airtime"
-		end
+  def getAirtimeUrl
```

Comment:
since these methods naming don't follow the Ruby conventions, maybe it would be better to "Ruby-ize" them, and then alias them for backward compatibility?
And at some point in the future, remove the aliases in a new major version of the SDK...

i.e. something like that:
```rb
def get_airtime_url
  getApiHost + '/version1/airtime'
end

alias_method :getAirtimeUrl, :get_airtime_url # deprecated

Reply:
Nice idea. :+1: